lib/content/message/predictions.ex

```diff
@@ -125,13 +146,34 @@ defmodule Content.Message.Predictions do
 
       track_number = Content.Utilities.stop_track_number(stop_id)
 
-      if track_number do
-        [
-          {Content.Utilities.width_padded_string(headsign, duration_string, width), 3},
-          {Content.Utilities.width_padded_string(headsign, "Trk #{track_number}", width), 3}
-        ]
-      else
-        Content.Utilities.width_padded_string(headsign, duration_string, width)
+      cond do
+        station_code == "RJFK" and destination == :alewife and zone == "m" ->
+          platform_name = Content.Utilities.stop_platform_name(stop_id)
+
+          {headsign_message, platform_message} =
+            if minutes == :max_time,
+              do: {headsign, " (Platform TBD)"},
+              else:
+                {headsign <> " (#{String.slice(platform_name, 0..0)})",
+                 " (#{platform_name} plat)"}
+
+          [
+            {Content.Utilities.width_padded_string(
+               headsign_message,
+               "#{duration_string}",
+               width
+             ), 3},
+            {headsign <> platform_message, 6}
```

Comment:
Revert of the revert, except with this value increased to 6.

The reason for this increase of the page duration is because ARINC suspects that the original feature was causing the sign to go offline because there was too much data going to the sign and its buffer was overflowing. Decreasing the frequency of the paging should help this.